GREEDY/Controllers/CreateAPathforDataController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using GREEDY.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.Controllers
+{
+    class CreateAPathforDataController
```

Comment:
In my opinion the proper name for this class would be CreatePathForDataController (The class name shouldn't use an article to make it shorter, and the word for should start with upper case)

Reply:
thanks. File was updated 👍